R/data_zones_to_areas.R

```diff
@@ -0,0 +1,34 @@
+# libraries ----
+library(sf)
+library(tidyverse)
+
+# load data ----
+zones<- st_read(here::here("data", "helmet_zones_map.shp"))
```

Comment:
A question: Is this input file `helmet_zones_map.shp` the same as the basic `aluejaot_2019_SHP/sijoittelualueet2019.shp`? Or is it something else?

Reply:
Almost! Shapefile `helmet_zones.shp` is sort of tidy version of `sijoittelualueet2019.shp`. Geometries are similar, but there is only one unique id `zone`, which corresponds to Helmet model system naming. Also it has attributes `area` and `municipality` that are found in Helmet.

And `helmet_zones_map.shp` is version where water areas are cut from the map, so plotting looks tidy without adding separate layers. This probably can be replaced later by using basemap layers.

Reply:
@atte-wsp Thanks for clarifying! 👍 Are the files in Sharepoint (`Saavutettavuuden jakautuminen/data`)?

Reply:
Yep, I stored the files temporary there. Let me know if there's a better place to store those.